docs/site/Loopback-component-authorization.md

```diff
@@ -0,0 +1,547 @@
+---
+lang: en
+title: 'Authorization'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Loopback-component-authorization.html
+---
+
+## Overview
+
+In every web application, we need to have a way to identify access rights of a
+user for any resource, which is known as **Authorization**. This is a
+minimalistic guide for creating such an implementation using Loopback component.
+This can be part of your main REST Application project or can be created as a
+Loopback extension for reuse in multiple projects. Latter is the better option
+for obvious reasons - reusability.
+
+**Note: Loopback team is working on making authorization an out-of-the-box
```

Comment:
Nitpick: `Loopback` -> `LoopBack`

Suggestion:
```
{% include note.html content="
The LoopBack team is working on making authorization an out-of-the-box
feature in LoopBack 4. It is a work in progress and will soon be there. Until
then, this implementation guide can be followed.
" %}
```

Reply:
Suggestion accepted and added.